src/js/views/maps/viewfinder/ExpansionPanelView.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+define(
+  [
+    'underscore',
+    'backbone',
+    'text!templates/maps/viewfinder/expansion-panel.html',
+  ],
+  (_, Backbone, Template) => {
+    // The base classname to use for this View's template elements.
+    const BASE_CLASS = 'expansion-panel';
+
+    /**
+     * @class ExpansionPanelView
+     * @classdesc Allow expand and collapse content in a panel.
+     * @classcategory Views/Maps/Viewfinder
+     * @name ExpansionPanelView
+     * @extends Backbone.View
+     * @screenshot views/maps/viewfinder/ExpansionPanelView_closed.png
+     * @screenshot views/maps/viewfinder/ExpansionPanelView_open.png
+     * @since x.x.x
+     * @constructs ExpansionPanelView
+     */
+    var ExpansionPanelView = Backbone.View.extend({
+      /**
+       * The type of View this is
+       * @type {string}
+       */
+      type: 'ExpansionPanelView',
+
+      /**
+       * The HTML class to use for this view's outermost element.
+       * @type {string}
+       */
```

Comment:
I think because we have `@extends Backbone.View`, you can simply use `@inheritdoc` for any methods are part of the base backbone view. I realize we're really not making use of this in older views in MetacatUI, but thought you should know about that shortcut.

Reply:
Done